pylossless/pipeline.py

```diff
@@ -414,8 +366,12 @@ def _get_outliers_quantile(array, dim, lower=0.25, upper=0.75, mid=0.5, k=3):
     """
     lower_val, mid_val, upper_val = array.quantile([lower, mid, upper],
                                                    dim=dim)
-    inter_q = upper_val - lower_val
-    return mid_val - inter_q*k, mid_val + inter_q*k
+
+    # Code below deviates from Tukeys method (Q2 +/- k(Q3-Q1))
```

Comment:
Leaving comment on _get_quantile method fix and deviation from TUKEYS method as requested...

Reply:
Some of my code-comment is not viewable right here (because they comment box is blocking it) - I think I put almost exactly what you said. I just want to confirm that you saw the whole comment.

Reply:
Here's the comment unobstructed so that it is easier to view.

https://github.com/lina-usc/pylossless/blob/56804ca4d6f02bbd3d8850c50e3fa1bbb75f68a2/pylossless/pipeline.py#L370-L371